lib/perl/Genome/Model/Tools/DetectVariants2/SpeedseqSv.pm

```diff
@@ -0,0 +1,109 @@
+#SpeedseqSv.pm
+
+
+package Genome::Model::Tools::DetectVariants2::SpeedseqSv;
+use warnings;
+use strict;
+
+use Genome;
+use File::Basename qw(basename dirname);
+use File::Spec;
+use IPC::System::Simple;
+use Genome::File::Tsv;
+
+class Genome::Model::Tools::DetectVariants2::SpeedseqSv {
+    is => 'Genome::Model::Tools::DetectVariants2::Detector',
+};
+
+	my $temp_directory = Genome::Sys->create_temp_directory();
+	my $pkg = 'Genome::Model::Tools::Speedseq::Sv';
```

Comment:
(Since it's only used once, it could also be taken out, and `Genome::Model::Tools::Speedseq::Sv->create` could be called directly.  We usually only use this `$pkg` shortcut in tests.)